doc/nrf/links.txt

```diff
@@ -50,6 +51,14 @@
 
 .. _`Ethernet over RTT for Linux`: https://github.com/doki-nordic/experimental-eth-rtt-link/
 
+.. _`Memfault firmware SDK`: https://github.com/memfault/memfault-firmware-sdk
+.. _`Memfault entry in the manifest`: https://github.com/nrfconnect/sdk-nrf/blob/20f40501f69bc9bfd2b321704917da1769411936/west.yml#L170-L173
+.. _`ANT entry in the manifest`: https://github.com/nrfconnect/sdk-nrf/blob/20f40501f69bc9bfd2b321704917da1769411936/west.yml#L182-L187
+
+.. _`Golioth Zephyr SDK`: https://github.com/golioth/golioth-zephyr-sdk
+.. _`Golioth module.yml`: https://github.com/golioth/golioth-zephyr-sdk/blob/main/zephyr/module.yml
+.. _`Golioth west-ncs.yml`: https://github.com/golioth/golioth-zephyr-sdk/blob/main/west-ncs.yml
```

Comment:
```suggestion
.. _`Golioth module.yml`: https://github.com/golioth/golioth-zephyr-sdk/blob/v0.5.0/zephyr/module.yml
.. _`Golioth west-ncs.yml`: https://github.com/golioth/golioth-zephyr-sdk/blob/v0.5.0/west-ncs.yml
```
Might be better to link files from latest stable release. No plans to change that now in `main`, if that would happen, at least this page does not get outdated, by using broken links. Links to memfault also use specific revision (by github permalinks?).

Reply:
Good point, fixed!